src/GooglePlacesApi.Abstractions/Interfaces/IGooglePlacesApi.cs

```diff
@@ -9,7 +10,10 @@ public interface IGooglePlacesApi
         [Get("/maps/api/place/autocomplete/json?input={input}")]
         Task<Predictions> GetAutocompleteAsync(string input, GoogleApiQueryStringParameters queryStringParameters);
 
-        [Get("/maps/api/place/details/json?key={apiKey}&placeid={placeId}")]
-        Task<Details> GetDetailsAsync(string apiKey, string placeId);
+        [Get("/maps/api/place/details/json?key={apiKey}&placeid={placeId}&fields={fields}")]
+        Task<Details> GetDetailsAsync(string apiKey, string placeId, string sessionToken, string fields);
+
+        [Get("/maps/api/place/photo?maxwidth=400&photoreference={photoReference}&key={apiKey}")]
```

Comment:
Is there a reason to use a maxwith of 400? Would it not be better to make it an (optional) parameter?

Reply:
True, forgot to look the width and height parameters. I'll implement them both with the default value of 1600 (is the maximum)